src/core/api/project.go

```diff
@@ -212,14 +215,16 @@ func (p *ProjectAPI) Post() {
 		return
 	}
 
-	quotaMgr, err := quota.NewManager("project", strconv.FormatInt(projectID, 10))
-	if err != nil {
-		p.SendInternalServerError(fmt.Errorf("failed to get quota manager: %v", err))
-		return
-	}
-	if _, err := quotaMgr.NewQuota(hardLimits); err != nil {
-		p.SendInternalServerError(fmt.Errorf("failed to create quota for project: %v", err))
-		return
+	if config.QuotaPerProjectEnable() {
+		quotaMgr, err := quota.NewManager("project", strconv.FormatInt(projectID, 10))
+		if err != nil {
+			p.SendInternalServerError(fmt.Errorf("failed to get quota manager: %v", err))
+			return
+		}
+		if _, err := quotaMgr.NewQuota(hardLimits); err != nil {
```

Comment:
If we don't add the quota and usage record into the database for the project, does the quota flow work well after we enabling the quota again?

Reply:
It will work well because quota manager will create the quota and quota usage for the project using default quota settings when they are not in db.

Reply:
for the scenario, disabel -> enable, it will call the sync quota to sync all of the usages.

@heww, I will remove the quota and usage adding from the DB scheme script and leave the library only. All of the others(projects) will be handled by the sync api. 

You can see the sync PR for details.